README.md

```diff
@@ -24,6 +24,57 @@ The goal to implement as many widgets as we can and make them customizable.
 
 *(These are not in a specific order)*
 
+## Getting started
+These instructions will walk you through how to get the project up and running on your machine for development and testing purposes.
+
+### Prerequisites
+These are the tools you need to build the extension
+
+- Latest version 14 of Node.js
```

Comment:
Understand what you are trying to convey but maybe it word it differently? Maybe "Latest node.js version (14.x.x.)"